Cargo.toml

```diff
@@ -35,4 +35,4 @@ version = "^0.1"
 optional = true
 
 [target.wasm32-unknown-unknown.dev-dependencies]
-wasm-bindgen-test = "=0.2.16"
+wasm-bindgen-test = "=0.2.19"
```

Comment:
@alexcrichton you meant like here?

Reply:
or do i need to require wasm-bindgen as well?

Reply:
Ah nah I think what's happening is this is locking `wasm-bindgen-test` but the dependency on `wasm-bindgen` is still floating